src/aosm/azext_aosm/definition_folder/reader/bicep_definition.py

```diff
@@ -153,6 +159,71 @@ def _artifact_manifests_exist(
 
         return ManifestsExist.NONE
 
+    @staticmethod

```

Comment:
Now that you have implemented the skipping base, can you remove the big comments from lines 232 to 237

Reply:
removed those lines